.eslintrc.js

```diff
@@ -1,22 +1,28 @@
 module.exports = {
   root: true,
 
-  plugins: ['import'],
-
-  extends: ['@metamask/eslint-config', '@metamask/eslint-config-nodejs'],
-
-  rules: {
-    'prefer-object-spread': 'off',
-  },
+  extends: ['@metamask/eslint-config'],
 
   overrides: [
     {
       files: ['*.ts'],
       extends: ['@metamask/eslint-config-typescript'],
     },
+
+    {
+      files: ['*.js'],
+      parserOptions: {
+        sourceType: 'script',
+      },
+      extends: ['@metamask/eslint-config-nodejs'],
+    },
+
     {
       files: ['test/*'],
       extends: ['@metamask/eslint-config-mocha'],
+      parserOptions: {
+        ecmaVersion: 2020,
```

Comment:
Hmm, good point, this won't get published. Should we add this in `@metamask/eslint-config-mocha` directly?

Reply:
Yeah, I guess so! I think there are scant places where we use it, and I remove it from this repo in #102, but why not!